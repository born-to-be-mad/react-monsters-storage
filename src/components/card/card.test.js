import { shallow } from "enzyme";
import React from "react";
import Card from "./card.component";

test("expect to render Card component", () => {
  expect(shallow(<Card />).length).toEqual(1);
});

test("expect to render Card component - snapshot", () => {
  const wrapper = shallow(<Card />);
  expect(wrapper).toMatchSnapshot();
});
