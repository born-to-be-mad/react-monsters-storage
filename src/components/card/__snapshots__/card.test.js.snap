// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component - snapshot 1`] = `
<div
  className="card-container1"
>
  <img
    alt="Monster"
    src="https://robohash.org/undefined?set=set2&size=180x180"
  />
  <h2 />
  <p />
</div>
`;
